ledger-tool/src/bigtable.rs

```diff
@@ -329,6 +379,39 @@ impl BigTableSubCommand for App<'_, '_> {
                                 .help("Maximum number of slots to return"),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("compare-blocks")
+                        .about("Find the missing confirmed blocks of an owned bigtable for a given range \
+                                by comparing to a reference bigtable")
+                        .arg(
+                            Arg::with_name("starting_slot")
+                                .validator(is_slot)
+                                .value_name("SLOT")
+                                .takes_value(true)
+                                .index(1)
+                                .required(true)
+                                .default_value("0")
+                                .help("Start listing at this slot"),
+                        )
+                        .arg(
+                            Arg::with_name("limit")
+                                .validator(is_slot)
+                                .value_name("LIMIT")
+                                .takes_value(true)
+                                .index(2)
+                                .required(true)
+                                .default_value("1000")
+                                .help("Maximum number of slots to return"),
+                        ).arg(
+                            Arg::with_name("reference_credential")
+                                .long("reference-credential")
+                                .short("f")
```

Comment:
This short isn't instinctive to me. Maybe `C`? Or no short?

Reply:
change to  "c". As a user, I want to have a short. xD